rules/network/zeek/zeek_http_sus_file_ext_from_sus_tld.yml

```diff
@@ -0,0 +1,98 @@
+title: HTTP Request to Low Reputation TLD or Suspicious File Extension
+id: 68c2c604-92ad-468b-bf4a-aac49adad08c
+status: test
+description: Detects HTTP requests to low reputation TLDs (e.g. .xyz, .top, .ru) or ending in suspicious file extensions (.exe, .dll, .hta), which may indicate malicious activity.
+references:
+    - https://www.howtogeek.com/137270/50-file-extensions-that-are-potentially-dangerous-on-windows/?utm_source=chatgpt.com
+    - https://www.spamhaus.org/reputation-statistics/cctlds/domains/
+author: '@signalblur, Corelight'
+date: 2025-02-26
+modified: 2025-02-26
+tags:
+    - attack.initial-access
+    - attack.command-and-control
+logsource:
+    product: zeek
+    service: http
+detection:
+    # Suspicious TLD in the 'host' field OR malicious file extension in the 'uri' field.
+    suspicious_tld:
+        host|endswith:
+            - '.xyz'
+            - '.top'
+            - '.ru'
+            - '.su'
+            - '.pw'
+            - '.ml'
+            - '.ga'
+            - '.cf'
+            - '.tk'
+            - '.gq'
+            - '.bid'
+            - '.work'
+            - '.party'
+            - '.click'
+            - '.loan'
+            - '.kp'
+            - '.by'
+            - '.mm'
+            - '.ir'
+            - '.sy'
+            - '.ve'
+            - '.cm'
+            - '.tv'
+    malicious_ext:
+        uri|endswith:
+            - '.exe'
+            - '.dll'
+            - '.hta'
+            - '.bat'
+            - '.cmd'
+            - '.vbs'
+            - '.js'
+            - '.ps1'
+            - '.jar'
+            - '.scr'
+            - '.pif'
+            - '.iso'
+            - '.reg'
+            - '.cpl'
+            - '.bin'
+            - '.msi'
+            - '.elf'
+            - '.py'
+            - '.sh'
+            - '.so'
+            - '.dylib'
+            - '.lnk'
+            - '.wsf'
+    malicious_mime:
+        resp_mime_types:
+            - 'application/x-dosexec'
+            - 'application/x-msdownload'
+            - 'application/x-msdos-program'
+            - 'application/vnd.microsoft.portable-executable'
+            - 'application/x-sh'
+            - 'application/x-python-code'
+            - 'application/x-bat'
+            - 'application/x-ms-shortcut'
+            - 'application/x-iso9660-image'
+            - 'application/x-java-archive'
+            - 'application/x-elf'
+
+    condition: suspicious_tld and (malicious_ext or malicious_mime)
+fields:
+    - ts
+    - uid
+    - id.orig_h
+    - id.resp_h
+    - id.resp_p
+    - host
+    - uri
+    - method
+    - status_code
+    - user_agent
+    - resp_mime_types
```

Comment:
```suggestion
```
The purpose of fields was to help a new analyst to start investigation. 
If you wan to keep `fields`, just put some.